src/node.h

```diff
@@ -335,6 +335,9 @@ NODE_DEPRECATED("Use DecodeWrite(isolate, ...)",
   return DecodeWrite(v8::Isolate::GetCurrent(), buf, buflen, val, encoding);
 })
 
+
+void PrintSyncWarning(Environment* env);
```

Comment:
@bnoordhuis I'm pretty sure `node.h` isn't the correct place for this, but wasn't sure where else to place it.